src/local-git/GitLocalInterface.ts

```diff
@@ -1,5 +1,6 @@
 import { Commit, Repository, NavigatorBackend } from "../NavigatorBackendType";
-import nodegit from "nodegit";
+import nodegit, { TreeEntry } from "nodegit";
```

Comment:
Seems like `TreeEntry` isn't used

```suggestion
import nodegit from "nodegit";
```